modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ItemActionForm.tsx

```diff
@@ -319,16 +433,123 @@ const ItemActionForm = ({
 											type: event.target.value,
 										})
 									}
-									options={TYPES}
+									options={
+										!activeTab
+											? ITEM_ACTION_TYPES
+											: CREATION_ACTION_TYPES
+									}
 									placeholder={Liferay.Language.get(
 										'please-select-an-option'
 									)}
 									value={actionData.type}
 								/>
 							</ClayForm.Group>
 						</ClayLayout.Col>
+
+						{actionData.type === ACTION_TYPE.MODAL && (
+							<ClayLayout.Col size={4}>
+								<ClayForm.Group>
+									<label htmlFor={typeFormElementId}>
+										{Liferay.Language.get('variant')}
+
+										<RequiredMark />
```

Comment:
Why do we want to mark Type and Variant as required when they do not allow empty values?

Reply:
:facepalm: Just seen that the bottom part of the UI is empty until the user selects an action type

![image](https://github.com/liferay-frontend/liferay-portal/assets/132653342/9b40a16d-e455-4977-ad67-d33d20e22869)

Reply:
> Why do we want to mark Type and Variant as required when they do not allow empty values?

Yes, see [comment in slack discussion](https://liferay.slack.com/archives/C3JBR21HA/p1695215103316539?thread_ts=1695211688.458709&cid=C3JBR21HA). cc @ugeortiz

Reply:
> Just seen that the bottom part of the UI is empty until the user selects an action type

`link` should be set by default.

Reply:
`link` is the default value for Item & Creation actions, showing the associated controls.